doc/advanced-configuration/Modules.md

```diff
@@ -80,17 +106,22 @@ This module implements [XEP-0016: Privacy Lists)](http://xmpp.org/extensions/xep
 ### [mod_private](../modules/mod_private.md)
 Implements [XEP-0049 (Private XML Storage)](http://xmpp.org/extensions/xep-0049.html) to store and query private user data in XML format.
 
+### [mod_pubsub](../modules/mod_pubsub.md)
+This extension implements [XEP-0060 (Publish-Subscribe)](http://www.xmpp.org/extensions/xep-0060.html). It is a pluggable implementation, that uses behaviours provided by `node_*.erl` and `nodetree_*.erl` modules.
+
 ### [mod_push](../modules/mod_push.md)
 Implements [XEP-0357 Push Notifiactions](https://xmpp.org/extensions/xep-0357.html) to provide push notifications to clients that are temporary unavailable.
 
 ### [mod_push_service_mongoosepush](../modules/mod_push_service_mongoosepush.md)
 Handles push notifications generated by [mod_pubsub](../modules/mod_pubsub.md)'s `node_push` and passes them to [MongoosePush](https://github.com/esl/MongoosePush) service.
 
-
 ### [mod_register](../modules/mod_register.md)
 Implements [XEP-0077: In-Band Registration)](http://xmpp.org/extensions/xep-0077.html), that enables creating an account and changing the password once connected.
 This does not provide a solution to the forgotten password use case via SMS or email.
 
+### [mod_revproxy](../modules/mod_revproxy.md)
```

Comment:
mod_revproxy.md should start with "Module Description", rather than "Reverse Proxy" as all modules docs start the same.